client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-view/motion-detail-view.component.ts

```diff
@@ -302,16 +321,51 @@ export class MotionDetailViewComponent extends BaseMeetingComponent implements O
         const indexOfCurrent = this._sortedMotions.findIndex(motion => motion === this.motion);
         if (indexOfCurrent > 0) {
             this.previousMotion = this._sortedMotions[indexOfCurrent - 1];
+            this.substituteSurroundingMotions(indexOfCurrent, true);
```

Comment:
I guess it would be better to just set the correct value instead of correcting it afterwards.